src/Authentication/Authentication/build-module.ps1

```diff
@@ -92,7 +92,7 @@ Copy-Item -Path "$cmdletsSrc/$ModulePrefix.$ModuleName.psd1" -Destination $outDi
 Copy-Item -Path "$cmdletsSrc/StartupScripts" -Filter *.ps1 -Recurse -Destination $outDir
 
 # Copy custom commands
-Copy-Item -Path "$cmdletsSrc/custom" -Filter *.ps1 -Recurse -Destination $outDir
+Copy-Item -Path "$cmdletsSrc/custom" -Recurse -Destination $outDir
```

Comment:
I don't think this is 100% safe -- you can easily pick up temp files left by editors for instance. If the goal is to include the test ps1 file, just add another `copy-item` line referencing the directory that hosts the files -- limiting the file type will keep you from picking up unwanted files. That said, I'm not sure what file(s) we're trying to grab here...

Reply:
This was to include the json file in the build process

Reply:
I noticed an error in the build while I tried to add a new Copy-Item line which is why I opted to omitting it

Reply:
Can we drill in to what the error was? We should fix that as the approach here could include unwanted files.